doc/RecordingControl.xml

```diff
@@ -2119,13 +2124,151 @@ secfrac = "." 1*6DIGIT</programlisting>
         </variablelist>
       </para>
       <para>
-        The device shall create an individual initialization vector for each segment.
-        The device shall encrypt with the <literal>Key</literal> configured for the encryption entry as the encryption key.
-        The device shall interpret the <literal>KID</literal> configured for the encryption entry as a 16-byte hexadecimal value and use this value as the key identifier.
+        <literal>Key/KID</literal> and <literal>AsymmetricEncryption</literal> elements of the encryption entry are mutually exclusive.
```

Comment:
This sentence comes a bit out of the blue. Suggest to move it to the asymmetric section.;